__tests__/Exciter.js

```diff
@@ -751,7 +751,7 @@ test('normalizeDataValues', (done) => {
       userId: '345',
       nestedMap: { someOther: 'thing' },
     },
-    skipMe: null,
+    skipMe: undefined,
```

Comment:
I swapped `undefined` in for `null` in several tests to ensure the functionality still works